src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const heads = table.tHead.rows;
+const rows = table.tBodies[0].rows;
+const foots = table.tFoot.rows;
```

Comment:
You don't need all these values 
Just use `document.querySelectorAll('tr')` to get all rows